setup.py

```diff
@@ -541,6 +544,7 @@ def do_setup():
             'docker': docker,
             'druid': druid,
             'elasticsearch': elasticsearch,
+            'exasol': exasol,
```

Comment:
Can you also add this package to the documentation?
https://airflow.readthedocs.io/en/latest/installation.html#extra-packages